API.md

```diff
@@ -0,0 +1,49 @@
+<pre>Activities</pre>
+title | CharField  
+subtitle | CharField  
+description | TextField  
+image_url | UrlField  
+image_retina_url | UrlField  
+url | UrlField  
+difficulty | CharField (choices)  
+duration | CharField (choices)  
+**Skills** | *:*  
+**WebLitSkills** | *:*  
+**Topic** | 1:*  
```

Comment:
If we're thinking "Read", "Write", and "Participate" would be the topics, I think we can probably scrap that. We only used that for the Web Lit Basics modules, and it's been difficult to group activities into those categories for all of the modules we've created since. I also don't think there's much user value in those topics.